pkg/platform/abstract/platform.go

```diff
@@ -402,6 +404,48 @@ func (ap *Platform) CreateProject(createProjectOptions *platform.CreateProjectOp
 	return platform.ErrUnsupportedMethod
 }
 
+// EnrichCreateProjectConfig enrich project configuration with defaults
+func (ap *Platform) EnrichCreateProjectConfig(createProjectOptions *platform.CreateProjectOptions) error {
+
+	// transform display name if needed
+	if !createProjectOptions.SkipTransformDisplayName {
+		if createProjectOptions.ProjectConfig.Spec.DisplayName != "" {
+
+			// name is UUID
+			if _, err := uuid.ParseUUID(createProjectOptions.ProjectConfig.Meta.Name); err == nil {
+				ap.transformDisplayNameToName(createProjectOptions.ProjectConfig)
```

Comment:
According to https://github.com/nuclio/nuclio/pull/1971/files#diff-1a490ff605cdf263dba98a3b457bbd69a2a61b363435e88509385488107a4ab2R256
you should transform if display name exists AND (name is uuid OR **name does not exist**)
I don't see you do it when name does not exist

Reply:
fixed